mloop/learners.py

```diff
@@ -1031,6 +1035,30 @@ def _rand2(self, index):
         '''
         r0, r1, r2, r3, r4 = self.random_index_sample(index, 5)
         return self.population[r0] + self.curr_scale * (self.population[r1] + self.population[r2] - self.population[r3] - self.population[r4])
+    
+    # custom evolutionary strategy
+    # may need to add a separate scaling factor
```

Comment:
Let's update the docstring to mention this instead of noting this in a comment. We should also try to make it easier to understand without much further reading. So the docstring should mention something along the lines of how we currently use the same scaling factor for "the vector from the current parameters to the best parameters" and "the vector(s) between other random population members" even though those could be different.